### ISSUE TITLE
[table]: page breaks when a table is rendered inside a collapsed panel

### ISSUE THREAD

**Issue by i832513** [SAP/ui5-webcomponents#8013]
### Bug Description

Try putting a table with some popin columns inside of a panel which is collapsed.

```html
  <ui5-panel
    header-text="Test"
    collapsed
  >
    <ui5-table>
      <ui5-table-row slot="default-1">
        <ui5-table-cell slot="default-1">
          <span>Notebook Basic 15</span>
        </ui5-table-cell>
        <ui5-table-cell slot="default-2">
          <span>Very Best Screens</span>
        </ui5-table-cell>
        <ui5-table-cell style="text-align: right" slot="default-3">
          <span>30 x 18 x 3cm</span>
        </ui5-table-cell>
        <ui5-table-cell style="text-align: right" slot="default-4">
          <span style="color: #2b7c2b">
            <b style="padding-right: 0.125rem">4.2</b>
            KG
          </span>
        </ui5-table-cell>
        <ui5-table-cell style="text-align: right" last-in-row="" slot="default-5">
          <span>
            <b style="padding-right: 0.125rem">956</b>
            EUR
          </span>
        </ui5-table-cell>
      </ui5-table-row>
      <ui5-table-column slot="columns-1" popin-display="Inline">
        <span>Product</span>
      </ui5-table-column>
      <ui5-table-column
        slot="columns-2"
        min-width="600"
        popin-text="Supplier"
        demand-popin=""
        popin-display="Inline"
      >
        <span>Supplier</span>
      </ui5-table-column>
      <ui5-table-column
        slot="columns-3"
        min-width="800"
        popin-text="Dimensions"
        demand-popin=""
        class="table-header-text-alignment"
        popin-display="Inline"
      >
        <span>Dimensions</span>
      </ui5-table-column>
      <ui5-table-column
        slot="columns-4"
        min-width="800"
        popin-text="Weight"
        demand-popin=""
        class="table-header-text-alignment"
        popin-display="Inline"
      >
        <span>Weight</span>
      </ui5-table-column>
      <ui5-table-column
        slot="columns-5"
        class="table-header-text-alignment"
        popin-display="Inline"
      >
        <span>Price</span>
      </ui5-table-column>
    </ui5-table>
  </ui5-panel>
```

### Affected Component

table

### Expected Behaviour

The panel should render collapsed, and the table will be hidden until the panel is expanded.

### Steps to Reproduce

Put a table with popin columns into a panel that is collapsed, or inside of a collapsible one which then is collapsed by click - both have the issue.

### Log Output, Stack Trace or Screenshots

![image](https://github.com/SAP/ui5-webcomponents/assets/137430976/1f61e0df-4ed8-4503-81a5-6326d69281e1)

### Priority

High

### UI5 Web Components Version

1.20.0

### Browser

Chrome

### Additional Context

I was not able to create an isolated example due to issues in codesandbox issues. This is reproduced on my local machine though, so it's not clear if this is somehow related to my usage or not.

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by nnaydenow** [comment]
Hi @i832513,

Could you please check if the issue is reproducible in the provided codesandbox and everything is configured correctly?
https://codesandbox.io/s/ui5-webcomponents-forked-3vdhqw?file=/index.html

I'm not able to reproduce the issue.

**Comment by i832513** [comment]
@nnaydenow ,

The issue is that "codesandbox" pages, at least for me, are all failing to load. I asked one other person to check and they are getting the same thing. There might be something I'm doing a little bit differently inside my own project - or it could be because I'm using the "webcomponents-react" and not plain web components. Either way, this issue seems clear cut that it only happens when the table is inside a Panel that is collapsed.

<img width="1693" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/137430976/5e578411-401f-4a31-a57d-17d85abddef1">

**Comment by petyabegovska** [comment]
Hi @i832513, 

For the Sandbox, can you check this proposal:
"if you’re on a SAP managed device, the cloud codeSandbox doesn’t work anymore… (Probably because of some kind of DNS blocking)
You can use the “old” one instead: https://codesandbox.io/s/ui5-web-components-for-react-vite-template-rcpmd4?file=/src/App.jsx
If you still face an error there, then codeSandbox added you to the beta. You can opt-out, by visiting your profile (https://codesandbox.io/u/<username>) and disabling it in the settings."

**Comment by ilhan007** [comment]
Hello colleagues,

I have created this Stackblitz sample, using the code @i832513 provided
https://stackblitz.com/edit/js-vsrpnb?file=index.js,package.json


The Panel renders collapsed and when I press to expand it, I can see the Table. 
@i832513 could you try opening the stackblitz, it should open for you

**Comment by i832513** [comment]
@ilhan007 ,
Stackblitz is working great.
https://stackblitz.com/edit/js-w29dfl?file=index.html,index.js

The problem is not always reproduced, apparently. You can play around with the above one and it will happen sometimes randomly. But here are some steps to reproduce it reliably:

1. Reduce the size of the frame, if necessary, so that only 2 columns of the table are visible
2. Now increase the size of the frame so that 3 columns are now visible
3. Click the panel to collapse

The frame should now break.

**Comment by ilhan007** [comment]
Hello Scott @i832513
I spent some time today to reproduce it, but unsuccessfully. I followed your steps, also tried other interactions to break it.
Several other colleagues could not reproduce it as well and unfortunately we can't continue with fixing this.

This is some part of my attempts:
https://github.com/SAP/ui5-webcomponents/assets/15702139/ac866e85-0be3-4cf1-a0de-3912737aa53e

If you find more reliable steps, please let us know..

BR,
ilhan

**Comment by i832513** [comment]
@ilhan007 ,
I too cannot reproduce the problem. I tried also on the widget usage where I was getting the problem and could not reproduce there either. I suspect maybe the chrome version I am using is different since it was so reliable before but now I seem to do be doing the same things as before and it is not happening.

**Comment by allen138** [comment]
@ilhan007 , I can consistently reproduce it. Please see:
https://github.com/SAP/ui5-webcomponents/assets/38701667/235e8f17-3855-48e1-9afd-59d668a57ea4

**Comment by i832513** [comment]
Note: When I attempted to reproduce the problem, I was sticking closely to the script in my comment from 3 weeks ago - the page starts with 2 columns then I expand the frame to have 3 and then collapse just like in @allen138 video link. But it was not reproduced for me. I do remember clearly, though, that doing these same steps (or as closely as i can remember) 3 weeks ago I was able to reproduce the problem. Though I did not record what version of chrome i was using at the time; however, i was using this same laptop (though I might have upgraded mac OS since then).

**Comment by hristop** [comment]
Hi @allen138 , @i832513 ,

We have tested this on several Macs with different versions of Sonoma 14.X.X (14.1.1 to 14.2.1) + several Chrome Version 120.0.6099.216 (Official Build) (arm64).
We even had a colleague with the same combination of OS + Browser:
 * Chrome: 120.0.6099.199 (Official Build) (arm64)
 * MacOS: 14.1.2 (Apple M2 Pro)

Also we have tested it on several Windows machines + the latest Chrome browser.

We have tried to reproduce the issue in several ways (not only by the specified steps above) but with no success. 

Can you please let us know if this is real use case of your application and does it affect a lot of users or it is simply something you have found? We are asking as we think that this may have had something to do with particular combination of OS and browser and it might have been resolved with some of the latest updates of either one of them (or combination - we can't be sure).

Have you tried restarting / updating to latest versions of chrome and Mac OS (with restarting after the update)?

Best Regards,
Hristo

**Comment by i832513** [comment]
@hristop ,

We do not have any current productive use cases where a Table with pop-in columns are put inside a Collapsible Panel. I noticed this with the mock data that I was using for testing purpose, and I was able to work around the problem by making sure the Panel was fixed and not collapsible in my test page.

Unfortunately, I'm no longer able to reproduce the problem - but I'll check with @allen138 to see if there is anything he can do to make the bug stop happening for him, such as restart his machine. I don't know why Allen can get it now but no one else can.
